tests/integration/test_truncate_database/test_replicated.py

```diff
@@ -50,7 +48,8 @@ def test_truncate_database_replicated(start_cluster):
     node1.query(
         "INSERT INTO test.test_table SELECT number, toString(number) FROM numbers(100)"
     )
-    assert node2.query("SELECT id FROM test.test_table LIMIT 1") == "0\n"
+    assert node2.query("SELECT min(id) FROM test.test_table") == "0\n"
```

Comment:
Even though change totally makes sense, it's unclear why if have been failing, since table has only one block of data 🤔

Reply:
We can wait until it fails again :smile:

Reply:
Found the culprit, a truncate database query was called between runs.